redis/src/test/scala/zio/redis/HashSpec.scala

```diff
@@ -58,6 +58,63 @@ trait HashSpec extends BaseSpec {
           } yield assert(deleted)(equalTo(2L))
         }
       ),
+      suite("hmSet and hmGet")(
+        testM("set followed by get") {
+          for {
+            hash   <- uuid
+            field  <- uuid
+            value  <- uuid
+            _      <- hmSet(hash, field -> value)
+            result <- hmGet(hash, field)
+          } yield assert(result.head)(isSome(equalTo(value)))
```

Comment:
I'd avoid unsafe calls, `headOption` or maybe even assert on the whole chunk is ok.